contracts/farm-bot.sol

```diff
@@ -11,6 +11,16 @@ import "./IRevoBounty.sol";
 import "./openzeppelin-solidity/contracts/AccessControl.sol";
 
 contract FarmBot is FarmbotERC20, AccessControl {
+    event BountyUpdated(address indexed by, address indexed to);
+    event SlippageUpdated(
+        address indexed by,
+        uint256 numerator,
+        uint256 denominator
+    );
+    event Deposit(address indexed by, uint256 lpAmount);
+    event Withdraw(address indexed by, uint256 lpAmount);
+    event ClaimRewards(address indexed by, uint256 lpStaked);
```

Comment:
Could it be useful to emit information about the fees paid out here? Don't want to overload this though.